draft-janfred-radext-radiusdtls-bis.md

```diff
@@ -183,26 +183,24 @@ While the client may be able to deduce the operational state of the local server
 Within RADIUS, proxies typically only forward traffic between the NAS and RADIUS servers, and they do not generate their own response.
 As a result, when a NAS does not receive a response to a request, this could be the result of packet loss between the NAS and proxy, a problem on the proxy, loss between the RADIUS proxy and server, or a problem with the server.
 
-When UDP is used as a transport protocol, the absence of a reply can cause a client to deduce (incorrectly) that the proxy is unavailable.
+The absence of a reply can cause a client to deduce (incorrectly) that the proxy is unavailable.
 The client could then fail over to another server or conclude that no "live" servers are available (OKAY state in {{!RFC3539}}, Appendix A).
 This situation is made even worse when requests are sent through a proxy to multiple destinations.
 Failures in one destination may result in service outages for other destinations, if the client erroneously believes that the proxy is unresponsive.
 
-For RADIUS/TLS, it is RECOMMENDED that implementations utilize the existence of a TCP connection along with the application-layer watchdog defined in {{RFC3539}}, Section 3.4 to determine that the server is "live".
-RADIUS/TLS clients MUST mark a connection DOWN, if the network stack indicates that the connection is no longer active.
-If the network stack indicates that the connection is still active, clients MUST NOT decide that it is down until the application-layer watchdog algorithm has marked it DOWN.
-RADIUS/TLS clients MUST NOT decide that a RADIUS/TLS server is unresponsive until all TLS connections to it have been marked down.[^contradiction]{:jf}
+It is REQUIRED that implementations utilize the existence of a TCP/DTLS connection along with the application-layer watchdog defined in {{RFC3539}}, Section 3.4 to determine that the server liveliness.
+RADIUS/TLS clients MUST mark a connection DOWN, under the following conditions:
```

Comment:
```suggestion
RADIUS/(D)TLS clients MUST mark a connection DOWN if one or more of the following conditions :
```

This is the suggestion from Stefan.

Since this is part of the general spec, it should be valid for both RADIUS/TLS and RADIUS/DTLS.